src/overrides/gemini-scrollbar.less

```diff
@@ -1,4 +1,5 @@
-@import (inline) "../../node_modules/react-gemini-scrollbar/node_modules/gemini-scrollbar/gemini-scrollbar.css";
+// Make sure to import the following path before including this file
+// @import (inline) "node_modules/react-gemini-scrollbar/node_modules/gemini-scrollbar/gemini-scrollbar.css";
```

Comment:
so the end user of `reactjs-components` would have to import the `gemini-scrollbar.css` correctly?